### ISSUE TITLE
ARIMA example: Error displaying widget: model not found

### ISSUE THREAD

**Issue by SeoliKim** [ARM-software/mango#104]
Thank you so much for this package. I am some trouble implementing it though. 

I followed the ARIMA model example. Everything was fine until "`results = tuner.minimize()`" where I received 
"**Error displaying widget: model not found**"

By the way, I am new to ARIMA model, what's the recommended range for param_space and num_iteration? 
![image](https://github.com/ARM-software/mango/assets/108114194/46cf1187-bac1-4caf-af2a-16c66b2cce82)

**Comment by sandeep-iitr** [comment]
Hi, Please see a complete example here to get an idea on tuning Arima: https://github.com/ARM-software/mango/blob/main/examples/timeseries/ARIMA_Example_1.ipynb

Also, there  is a blog written by me on the Arima model tuning which will give you a better idea of this notebook, and tuning parameters: https://medium.com/@sandha.iitr/tuning-arima-for-forecasting-an-easy-approach-in-python-5f40d55184c4

Basically, in your example, you are missing the Arima model import, something as below:

`from statsmodels.tsa.arima.model import ARIMA
`
You might have to install the library (statsmodels for ARIMA in your machine)
`pip install statsmodels
`

**Comment by SeoliKim** [comment]
Thank you so much for your rapid respond. In fact I have been following this example (https://github.com/ARM-software/mango/blob/main/examples/timeseries/ARIMA_Example_1.ipynb). 
 I have installed and imported statsmodel and mane other libraries (see the photo attached). 
As you can see in the my previous image, the program is able to process until `tuner = Tuner(param_space, arima_objective_function, conf_Dict)`, but not `results = tuner.minimize()`. Do you know why? 
![image](https://github.com/ARM-software/mango/assets/108114194/7f35babf-afee-49d0-b8bc-d0e333e6367b)

**Comment by SeoliKim** [comment]
Hi, could you send me a list of required packages and libraries for ARIMA model parameter tuning? I saw this link with the requirements listed (https://github.com/ARM-software/mango/blob/main/examples/requirements.txt), are they all needed for ARIMA model?

**Comment by SeoliKim** [comment]
Hi, thank you for your reply. I have successfully tuned the parameter
myself. I am not sure how to replicate the issue. Please close the thread.
Thank you for your offer to help.

Sincerely, Seoli Kim
